<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Curve</class>
 <widget class="QDialog" name="Curve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCustomPlot" name="curve_plot" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>341</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>61</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>X Point 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="x_point2">
      <property name="maximum">
       <double>255.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Y Point 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="y_point2">
      <property name="maximum">
       <double>255.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addpoint1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_clear">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>61</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>X Point 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="x_point1">
      <property name="maximum">
       <double>255.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Y Point 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="y_point1">
      <property name="maximum">
       <double>255.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
